### ISSUE TITLE
Crash on closing tab running powershell in CMD

### ISSUE THREAD

**Issue by legowerewolf** [microsoft/terminal#2394]
Feedback Hub link: https://aka.ms/AA5svj0

# Environment

```none
Windows build number: [10.0.18362.239]
Windows Terminal version (if applicable): 0.3.2171.0

Any other software? 
* Powershell 5.1.18362.145
```

# Steps to reproduce

1. Create a new cmd.exe tab.
2. Use command 'powershell'.
3. Close the tab.

# Expected behavior

The tab should close with no problems. 

# Actual behavior

Terminal crashes, bringing all other tabs with it.

**Comment by richardszalay** [comment]
I think this might be a duplicate of #2248

**Comment by legowerewolf** [comment]
@richardszalay - I don't think so. This has nothing to do with how fast you do it, and I'm pretty sure it has to do with nesting terminals.

**Comment by legowerewolf** [comment]
The link's there in the initial issue. I *did* read the template.

**Comment by DHowett-MSFT** [comment]
Thanks. Sorry I missed it.

**Comment by DHowett-MSFT** [comment]
It looks like we're deadlocking tearing down the connection (MSFT:22984264). Thanks!

**Comment by zadjii-msft** [comment]
This might be #1810 which is fixed by #2525. Hopefully.

**Comment by legowerewolf** [comment]
When might we see #2525 in a release?
